domain-cc/cc-app/src/python_src/util/lookup_table.py

```diff
@@ -144,6 +151,36 @@ def get_v1_lookup_table(filepath: str, input_key: str, output_key: str) -> dict:
     return classification_code_mappings
 
 
+class ContentionTextLookupTable:
+    CSV_FILEPATH = os.path.join(
+        os.path.dirname(__file__),
+        "data",
+        "condition_dropdown_lookup_table",
+        contention_lut_csv_filename,
+    )
+    input_key = "CONTENTION TEXT"
+    output_key = "CLASSIFICATION CODE"
+    classification_code_mappings = {}
+
+    def __init__(self):
+
+        with open(self.CSV_FILEPATH, "r") as fh:
+            csv_reader = csv.DictReader(fh)
+            for row in csv_reader:
+                table_key = row[self.input_key].strip().lower()
+                self.classification_code_mappings[table_key] = int(row[self.output_key])
+
+    def get(self, input_str: str, fallback=None):
+        input_str = input_str.strip().lower()
+        classification = self.classification_code_mappings.get(input_str, fallback)
+        print(f"(ContentionTextLookupTable) returning {classification} for {input_str}")
+        # return self.classification_code_mappings.get(input_str, fallback)
+        return classification
```

Comment:
improve readability by returning 
type Classification or a named tuple or _something_ that includes both `classification_code` and `classification_name`